Description: remove dependency on security-framework
 The dependency on security-framework is only necessary for macos.
Author: Jan Niehusmann <jan@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2021-05-12

--- rust-rustls-native-certs-0.5.0.orig/Cargo.toml
+++ rust-rustls-native-certs-0.5.0/Cargo.toml
@@ -37,7 +37,5 @@ version = "0.21"
 default = ["rustls"]
 [target."cfg(all(unix, not(target_os = \"macos\")))".dependencies.openssl-probe]
 version = "0.1.2"
-[target."cfg(target_os = \"macos\")".dependencies.security-framework]
-version = "2.0.0"
 [target."cfg(windows)".dependencies.schannel]
 version = "0.1.15"
